apps/payments/next/config.ts

```diff
@@ -0,0 +1,478 @@
+    doc: 'The current node.js environment',
+    env: 'NODE_ENV',
+    format: ['development', 'production', 'test'],
+  },
+  flow_id_expiry: {
+    default: '2 hours',
+    doc: 'Time after which flow ids are considered stale',
+    env: 'FLOW_ID_EXPIRY',
+    format: 'duration',
+  },
+  googleAnalytics: {
+    enabled: {
+      default: false,
+      doc: 'Toggle Google Analytics enabled',
+      env: 'GA_ENABLED',
+      format: Boolean,
+    },
+    measurementId: {
+      default: '',
+      doc: 'Google Analytics measurement ID',
+      env: 'GA_MEASUREMENT_ID',
+      format: String,
+    },
+    supportedProductIds: {
+      default: 'prod_GqM9ToKK62qjkK',
+      doc: 'Comma separated string of supported Stripe Product IDs',
+      env: 'GA_SUPPORTED_STRIPE_PRODUCT_IDS',
+      format: String,
+    },
+    testMode: {
+      default: false,
+      doc: 'Toggle Google Analytics test/debug mode',
+      env: 'GA_TEST_MODE',
+      format: Boolean,
+    },
+  },
+  geodb: {
+    dbPath: {
+      default: path.resolve(__dirname, '../../../fxa-geodb/db/cities-db.mmdb'),
+      doc: 'Path to maxmind database file',
+      env: 'GEODB_DBPATH',
+      format: String,
+    },
+    enabled: {
+      default: true,
+      doc: 'Feature flag for geolocation',
+      env: 'GEODB_ENABLED',
+      format: Boolean,
+    },
+    locationOverride: {
+      doc: 'override for forcing location',
+      format: Object,
+      default: {},
+      env: 'GEODB_LOCATION_OVERRIDE',
+    },
+  },
+  hstsEnabled: {
+    default: true,
+    doc: 'Send a Strict-Transport-Security header',
+    env: 'HSTS_ENABLED',
+    format: Boolean,
+  },
+  hstsMaxAge: {
+    default: 31536000, // a year
+    doc: 'Max age of the STS directive in seconds',
+    // Note: This format is a number because the value needs to be in seconds
+    format: Number,
+  },
+  legalDocLinks: {
+    privacyNotice: {
+      default: 'https://www.mozilla.org/privacy/firefox-private-network',
+      doc: 'Link to Privacy Notice',
+      env: 'PAYMENT_PRIVACY_NOTICE',
+      format: 'url',
+    },
+    termsOfService: {
+      default:
+        'https://www.mozilla.org/about/legal/terms/firefox-private-network',
+      doc: 'Link to Terms of Service',
+      env: 'PAYMENT_TERMS_OF_SERVCIE',
+      format: 'url',
+    },
+    cdnFqdn: {
+      default: 'accounts-static.cdn.mozilla.net',
+      doc: 'The domain name where the legal doc downloads are hosted.',
+      env: 'PAYMENT_LEGAL_DOWNLOAD_FQDN',
+      format: String,
+    },
+    httpResCacheLimit: {
+      default: 65536,
+      doc: 'The max number of entries in the redirect endpoint HTTP results cache.  0 means unlimited and the memory usage on the cache could reach the max of Map (1GB on V8)',
+      env: 'PAYMENT_LEGAL_DOWNLOAD_CACHE_LIMIT',
+      format: Number,
+    },
+  },
+  listen: {
+    host: {
+      default: '0.0.0.0',
+      doc: 'The ip address the server should bind',
+      env: 'IP_ADDRESS',
+      format: String,
```

Comment:
I think we can use `'ipaddress'` here, per https://github.com/mozilla/node-convict/tree/master/packages/convict-format-with-validator
```suggestion
      format: 'ipaddress',
```